boringssl-static/pom.xml

```diff
@@ -75,6 +75,282 @@
   </dependencies>
 
   <profiles>
+
+    <!-- The profile that builds a fips-boringssl-static jar -->
+    <profile>
+      <id>fips-boringssl-static</id>
+      <properties>
+        <boringsslCheckoutDir>${project.build.directory}/boringssl-${boringsslBranch}/boringssl</boringsslCheckoutDir>
+        <boringsslBuildDir>${boringsslCheckoutDir}/build</boringsslBuildDir>
+        <!--   Latest FIPS compliant boringSSL commit   -->
+        <boringsslBranch>${env.BORINGSSL_COMMIT}</boringsslBranch>
+        <linkStatic>true</linkStatic>
+        <msvcSslIncludeDirs>${boringsslCheckoutDir}/include</msvcSslIncludeDirs>
+        <msvcSslLibDirs>${boringsslBuildDir}/ssl;${boringsslBuildDir}/crypto;${boringsslBuildDir}/decrepit</msvcSslLibDirs>
+        <msvcSslLibs>ssl.lib;crypto.lib;decrepit.lib</msvcSslLibs>
+        <jniArch>${os.detected.arch}</jniArch>
+      </properties>
+
+      <build>
+        <plugins>
+
+          <!-- Download the BoringSSL source -->
+          <plugin>
+            <groupId>com.googlecode.maven-download-plugin</groupId>
+            <artifactId>download-maven-plugin</artifactId>
+            <version>1.6.8</version>
+            <executions>
+              <execution>
+                <id>install-fips-boringssl</id>
+                <phase>process-sources</phase>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <url>https://commondatastorage.googleapis.com/chromium-boringssl-fips/boringssl-853ca1ea1168dff08011e5d42d94609cc0ca2e27.tar.xz</url>
```

Comment:
How would we keep track of the right tar ball to download ?

Reply:
Updated in commit: d1e16bac4954fa2ad2ec9b17143232e256e42350